zulip_bots/zulip_bots/bots/merels/libraries/game.py

```diff
@@ -0,0 +1,211 @@
+"""This is the main component of the game, the core of the game that squishes
+everything together and make the game work. Usually user can just import this
+module and use the beat() function and everything will be fine, but will there
+be any certain things that can't be accomplished that way, the user may also
+freely import another modules.
+"""
+
+import re
+
+from . import mechanics
+
+COMMAND_PATTERN = re.compile(
+    "^(\\w*).*(\\d,\\d).*(\\d,\\d)|^(\\w+).*(\\d,\\d)")
+
+
+def getInfo():
+    """ Gets the info on starting the game
+
+    :return: Info on how to start the game
+    """
+    return "To start a game, mention me and add `create`. A game will start " \
+           "in that topic. "
+
+
+def getHelp():
+    """ Gets the help message
+
+    :return: Help message
+    """
+
+    return "Commands: create, reset, put (v,h), move (v,h) -> (v,h), take (" \
+           "v,h) "
+
+
+def unknown_command():
+    """Returns an unknown command info
+
+    :return: A string containing info about available commands
+    """
+    return "Unknown command. Available commands: create, reset, put (v,h), " \
+           "take (v,h), move (v,h) -> (v,h)"
+
+
+def beat(message, topic_name, merels_database):
+    """ This gets triggered every time a user send a message in any topic
+
+    :param message: User's message
+    :param topic_name: User's current topic
+    :param merels_database: MerelsDatabase object
+    :return: A response string to reply to that topic, if any. If not, it
+            returns an empty string
+    """
+
+    if "create" in message.lower():
+        return mechanics.create_room(topic_name, merels_database)
+
+    if "help" in message.lower():
+        return getHelp()
+
+    if "reset" in message.lower():
```

Comment:
The reset command didn't work in our tests: Even though we had a game started, it printed "No game created yet".

Reply:
Sorry, forgot to put `not` in `is None` below that. Just what happens when you write it at 3 AM. Should work fine now.